packages/core/src/services/chains/ChainErrors/index.ts

```diff
@@ -1,9 +1,10 @@
 import { RunError } from '../../../browser'
 import { RunErrorCodes, RunErrorDetails } from '../../../constants'
+import { LatitudeError, LatitudeErrorDetails } from '../../../lib'
 
-export class ChainError<T extends RunErrorCodes> extends Error {
+export class ChainError<T extends RunErrorCodes> extends LatitudeError {
```

Comment:
I'm inheriting from `LatitudeError` so when failing in the gateway users see this output
https://github.com/latitude-dev/latitude-llm/blob/main/apps/gateway/src/middlewares/errorHandler.ts#L30-L34

What do you think?